requirements.txt

```diff
@@ -3,7 +3,7 @@ aiohttp
 altair>=4.2.0
 fastapi
 ffmpy
-gradio_client>=0.0.10
+gradio_client==0.1.2b1
```

Comment:
Triple check this is the right version 😂

Reply:
Good catch!